src/systems.rs

```diff
@@ -0,0 +1,14 @@
+use specs::prelude::*;
+use crate::components::*;
```

Comment:
Put a line between this and the above use statement. To separate out external crates from crate-local use statements.